wafer/users/templates/wafer.users/registration_confirm_mail.txt

```diff
@@ -0,0 +1,12 @@
+{% load i18n %}
+{% blocktrans %}
+Thank you for registering for {{ conference_name }}.
```

Comment:
Because it isn't using the same context processors as the normal templates.

I wanted to make that clearer, to anyone who is editing them.

If you look at the registration (other registration) email templates, you'll see they live in a different world, too.